packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -48,6 +51,41 @@ class MemoryController extends DisposableController
 
   static const String logFilenamePrefix = 'memory_log_';
 
+  bool showHeatMap = true;
+
+  final List<Snapshot> snapshots = [];
+
+  Snapshot get lastSnapshot => snapshots.isNotEmpty ? snapshots.last : null;
+
+  /// Notifies that the source of the memory feed has changed.
+  ValueListenable get selectedSnapshotNotifier => _selectedSnapshotNotifier;
+
+  final _selectedSnapshotNotifier = ValueNotifier<String>('');
+
+  set selectedSnapshotTimestamp(String snapshotTimestamp) {
+    _selectedSnapshotNotifier.value = snapshotTimestamp;
+  }
+
+  String get selectedSnapshotTimestamp => _selectedSnapshotNotifier.value;
+
+  HeapGraph theGraph;
+
+  /// Leaf node of tabletree snapshot selected?  If selected then the instance
+  /// view is displayed to view the fields of an instance.
+  /// Notifies that the timeline has been paused.
+  final _leafSelectedNotifier = ValueNotifier<HeapGraphElementActual>(null);
+
+  ValueListenable get leafSelectedNotifier => _leafSelectedNotifier;
+
+  ValueNotifier<HeapGraphElementActual> get selectedLeaf =>
+      _leafSelectedNotifier;
+
+  bool get isLeafSelected => selectedLeaf.value != null;
+
+  void setSelectedLeaf(HeapGraphElementActual selected) {
```

Comment:
make this a setter instead of `setSelectedLeaf`

Reply:
Done.